tldr/cross_file_calls.py

```
@@ -3939,3 +3965,1024 @@ def _build_php_call_graph(
+        for caller, clist in calls.items():
+            for ctype, target in clist:
+                if ctype == "direct":
+                    if target in defs:
+                        graph.add_edge(rel, caller, rel, target)
+                    elif target in global_defs:
+                        graph.add_edge(rel, caller, global_defs[target], target)
+                elif ctype == "attr":
+                    if target in defs:
+                        graph.add_edge(rel, caller, rel, target)
+                    elif target in global_defs:
+                        graph.add_edge(rel, caller, global_defs[target], target)
```

Ruff (0.15.12)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)